### ISSUE TITLE
Unexpected token < in JSON at position 0

### ISSUE THREAD

**Issue by heyalbert** [netlify/netlify-cms#1455]
**- Do you want to request a *feature* or report a *bug*?**
bug

**- What is the current behavior?**
Error message when trying to log into the admin panel of my netlify website.

```
Unexpected token < in JSON at position 0
```

![Screenshot showing the bug](https://i.imgur.com/rN1TrNf.png)

**- If the current behavior is a bug, please provide the steps to reproduce.**
Created a fresh new netlify cms with gatsby using bitbucket.

1. app.netlify.com -> Identity
2. Invite users -> Email address
3. Inbox, clicked on "Accept the invite"
4. Typed in my new password.
5. Went through /admin and error message appeared. 

**- What is the expected behavior?**
Login/Signup into the admin panel of my new created netlifiy cms.

**Comment by tech4him1** [comment]
BitBucket is not yet supported by Netlify CMS. You can follow https://github.com/netlify/netlify-cms/issues/234 for updates!

**Comment by AlekseiAQ** [comment]
Same issue /w gitlab, but gitlab is supported, right?

**Comment by galvarez421** [comment]
I just had this same issue using GitLab repo + Netlify Identity. I didn't invite a user like the OP. I just created my own test account, got the email to confirm my email address, confirmed, then I was redirected to the login screen and got the same exact JSON error message.

The console error message says there is a 404 GET request error for retrieving https://[mysite].netlify.com/.netlify/git/settings.

The error message only appears if I am logged in as the test user. If I open the /admin/ URL in a fresh incognito window (not logged in), I don't see the message, but after I log in, it'll say I have logged in and nothing happens except it says I'm logged in and there is a button to log out or close the login confirmation modal. If I refresh the page, it'll show the error again.
